.github/workflows/new-milestone-and-issue-in-other-repo.yml

```diff
@@ -2,8 +2,8 @@
 # GitHub Action to Create New Milestone and issue in Other Repositories.
 #
 #############################################################################
-name: "R: Close/Create Milestone, Upload '-all' jar, Create issues"
-run-name: "R: Close/Create Milestone, Upload '-all' jar, Create issues ${{ inputs.version }}"
+name: "R: Close/Create Milestone, Create issues"
+run-name: "R: Close/Create Milestone, Create issues ${{ inputs.version }}"
```

Comment:
Is name and run-name basically the same, but one says what name it goes by under a run? If not, what is the difference and why do we have these as 2 different names?

Reply:
Name for action title.
Run-name is for execution instance

Reply:
> why do we have these as 2 different names?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#name

Reply:
Action is workflow. We have name for workflow, it is like a name of class. And we have name for execution (run-name), we make it same as name of workflow plus run parameters. Execution is like instance of workflow.

Reply:
But that doesn't say why they have to be different and can't be the same. For example, why aren't they both `"R: Close/Create Milestone, Create issues ${{ inputs.version }}"`. We didn't use to do this name and run-name in other workflows.

Reply:
`${{ inputs.xx }}` context is not supported in `name:`, only in `run-name`.

Reply:
@stoyanK7 Thanks. I did not know this.

Just to confirm, you are talking only about the top level name? I see sub-names have `${{}}`.

Reply:
Yes, the sub-level names are names for `job`s, not the workflow itself.

Reply:
Thanks.